exampleAppJapan/src/main/kotlin/it/trade/android/japanapp/MainActivity.kt

```diff
@@ -12,7 +12,7 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.main_activity)
         if (savedInstanceState == null) {
             supportFragmentManager.beginTransaction()
-                    .replace(R.id.container, OrderInputFragment.newInstance())
+                    .replace(R.id.container, OrderInputFragment.newInstance("8704"))
```

Comment:
Only for clarity, it might be worth to have `8704` in a variable with the name of the symbol.

Reply:
it's only for testing purpose. since in the final app, the Activity will be provided by Yahoo Japan. they should use Fragment directly.
I'll add a comment to this line. also for our example app, will need another sample fragment where user can choose the symbol to trade(symbolLookup? as in iOS example App)